RecoTracker/MkFitCore/interface/HitStructures.h

```diff
@@ -0,0 +1,874 @@
+#ifndef RecoTracker_MkFitCore_interface_HitStructures_h
+#define RecoTracker_MkFitCore_interface_HitStructures_h
+
+#include "RecoTracker/MkFitCore/interface/Config.h"
+#include "RecoTracker/MkFitCore/interface/Hit.h"
+#include "RecoTracker/MkFitCore/interface/Track.h"
+#include "RecoTracker/MkFitCore/interface/TrackerInfo.h"
+
+#include <algorithm>
+#include <array>
+#include "tbb/concurrent_vector.h"
+
+//#define DUMPHITWINDOW
```

Comment:
needed?

Reply:
Addressed in Round 2 follow-up PR trackreco/cmssw#76 commit https://github.com/trackreco/cmssw/pull/76/commits/11e374bd87f89c5f1e214ea7f252c0f8f096d401

removed

Reply:
Yes, these are used for tuning of hit-search windows.